libs/galley-types/src/Galley/Types/Teams.hs

```diff
@@ -655,3 +723,20 @@ instance ToJSON TeamDeleteData where
     toJSON tdd = object
         [ "password" .= _tdAuthPassword tdd
         ]
+
+#ifdef WITH_CQL
+instance Cql.Cql Permissions where
```

Comment:
not sure which place makes most sense given the related types, but sure, i can move it.  will do!

Reply:
what module should i move this to, then?  i can't think of any way to move it that i would prefer over the current code here, but i'm happy to implement a concrete suggestion.

Reply:
`Data.Misc` ?

Reply:
I decided I'm opposed to this change for now.  It becomes more and more obvious that brig and galley aren't really two different services and should be merged.  Until we solve that issue, I want to keep the galley types in their proper place, and not spread them out over `Data.Galley` and `Galley.Types.Team`.

Reply:
Following the above logic of service types dependencies, can we put this (`Permissions` and `Role`) under `Data.Role` on types-common? Even if `galley` and `brig` do get merged, at least until then let's keep things consistent;

Note that the whole invitation should and could be part of galley - the reason we decided against it was because all the email logic/templating/etc. was part of brig